Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -32,6 +32,7 @@
 # AdditionalEncoder - generate serializers for StreamConnectionEncoder in addition to IPC::Encoder.
 # CreateUsing - use a custom function to call instead of the constructor or create.
 # CustomHeader - don't include a header based on the struct/class name. Only needed for non-enum types.
+# DisableMissingAttributeCheck - do not check for attributes that are missed during serialization.
```

Comment:
DisableMissingMemberCheck?

Reply:
Ok, I'll update the name before landing.